lib/ansible/modules/monitoring/zabbix/zabbix_host_triggers.py

```diff
@@ -0,0 +1,254 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) stephane.travassac@fr.clara.net
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+RETURN = '''
+---
+output:
```

Comment:
Actually `output` field which contains `triggers_ok` and `triggers_problem` child elements is never returned. What you want here is having `triggers_ok` and `triggers_problem` fields at the root level with `type=json`.
Please see the documentation [here](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#return-block).

Reply:
@rubentsirunyan like:
```
RETURN = '''
---
triggers_ok:
    description: Host Zabbix Triggers in OK state
    returned: On success
    type: complex
    contains:
            - comments: Raise an alert whenever there is no data received from the agent during
                    the given interval
              description: 'Status {HOSTNAME}: server is unreachable'
              error: ''
              expression: "{13665}=1"
              flags: '0'
              lastchange: '1556112352'
              priority: '4'
              state: '0'
              status: '0'
              templateid: '13619'
              triggerid: '14038'
              type: '0'
              url: ''
              value: '0'
triggers_problem: ........
```

Reply:
yes, this looks better